requirements/software.md

```diff
@@ -0,0 +1,99 @@
+# Software Requirements
+
+* ### Pharmacy Files
+
+    It is likely that a software project has several different engineering
+    documents associated with it such as software requirements,
+    architecture design, and so forth. Each of these documents may have a
+    corresponding Rx file that Howser can use to validate their
+    conformance to a prescribed format. In a continuous integration
+    environment, it makes sense to have those validations be performed
+    automatically as a step in the build process. In order for that to
+    occur, there must be a way to specify which documents in the project
+    should be checked for conformance and which prescriptions each of
+    those documents should be checked against. This is the motivation for
+    introducing a `pharmacy.toml` file as a way to specify this configuration
+    on a per-project basis. Pharmacy files are written in toml and checked
+    into the repository along with the source code. They specify a series
+    of key value pairs that correspond to prescription files and the
+    documents that are to be checked with them.
+    
+    **Rx spec** = A DSL for constraining the content of Markdown 	documents
+
+    **prescription file (.rx)** =  A document written using the Rx spec DSL
+
+    **Pharmacy file (.toml)** = A list of documents to validate paired with the 	prescription they’re validated against
```

Comment:
Looks like there are some funny tabs/spacing included in this line and the `**Rx spec**` one above.